projects/packages/forms/src/blocks/input-phone/index.js

```diff
@@ -7,8 +7,11 @@ import save from './save';
 const name = 'phone-input';
 const settings = {
 	apiVersion: 3,
-	title: __( 'International Phone Input', 'jetpack-forms' ),
-	description: __( 'A compound input for international phone numbers.', 'jetpack-forms' ),
+	title: __( 'Phone Input', 'jetpack-forms' ),
+	description: __(
+		'A compound input for phone numbers with international support.',
```

Comment:
```suggestion
		'Phone number input.',
```

We could keep it simple?